docs/reference/mapping/removal_of_types.asciidoc

```diff
@@ -157,22 +157,24 @@ GET twitter/tweet/_search
 ----
 // NOTCONSOLE
 
+Note the use of the `include_type_name=true` url parameter on index creation.
+This is necessary as the use of types is deprecated in 7.0. In this example
```

Comment:
Small suggestion: instead of 'This is necessary as the use of types is deprecated in 7.0.', we could say 'This is necessary because index creation requests are typeless by default in 7.0.'

Reply:
Actually, now thinking about it, the example uses two types still and is only there to exemplify the "old" syntax. It shoudln't use "include_type_names" at all and is also not run in the tests (//NOTCONSOLE) so I think we should remove this note again and also remove the parameter here, since it can only work in 5.6 indices this  way.